### PR TITLE
INT-2230 - New properties added to `sentry_member`

### PR DESCRIPTION
**Pull request by austinkelleher**

### CODE REVIEW

File: src/steps/access/converter.ts [hunk 1]
```diff
@@ -46,9 +47,17 @@ export function createSentryUserEntity(user: SentryUser) {
         _key: `sentry-user-${user.id}`,
         _type: Entities.MEMBER._type,
         _class: Entities.MEMBER._class,
+        createdOn: parseTimePropertyValue(user.dateCreated),
         username: user.email,
         role: user.role,
         mfaEnabled: user.user.has2fa,
+        active: user.user.isActive === true,
```
Comment (austinkelleher) [inline src/steps/access/converter.ts:54]: @erichs

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,3 +7,36 @@ and this project adheres to
...
+
+- New properties added to resources:
+
+  | Entity          | Properties                                                                                            |
+  | --------------- | ----------------------------------------------------------------------------------------------------- |
+  | `sentry_member` | `createdOn`, `active`, `isManaged`, `isStaff`, `isSuperuser`, `dataJoined`, `lastActive`, `lastLogin` |
+
+## 1.0.0 - 2021-11-12
+
+### Added
```
Comment (austinkelleher) [inline CHANGELOG.md:21]: Bonus CHANGELOG updates

File: src/steps/access/converter.ts [hunk 3]
```diff
@@ -46,9 +47,17 @@ export function createSentryUserEntity(user: SentryUser) {
...
+        isManaged: user.user.isManaged === true,
+        isStaff: user.user.isStaff === true,
+        isSuperuser: user.user.isSuperuser === true,
+        dateJoined: parseTimePropertyValue(user.user.dateJoined),
+        lastActive: parseTimePropertyValue(user.user.lastActive),
+        lastLogin: parseTimePropertyValue(user.user.lastLogin),
```
Comment (austinkelleher) [inline src/steps/access/converter.ts:60]: Some additional properties that I think will be useful

File: src/steps/access/converter.ts [hunk 4]
```diff
@@ -46,9 +47,17 @@ export function createSentryUserEntity(user: SentryUser) {
         _key: `sentry-user-${user.id}`,
         _type: Entities.MEMBER._type,
         _class: Entities.MEMBER._class,
+        createdOn: parseTimePropertyValue(user.dateCreated),
```
Comment (austinkelleher) [inline src/steps/access/converter.ts:50]: @adam-in-ict `createdOn` is a standard property that we try to add to all entities that expose it. There is also `updatedOn`.

**Review by erichs — APPROVED** [review]

**Review by adam-in-ict — APPROVED** [review]

**Pull request merged**